src/databricks/labs/ucx/assessment/export.py

```diff
@@ -35,19 +35,32 @@ def export_results(self, prompts: Prompts):
         query_choice = prompts.question(
             "Choose which assessment results to export",
             default="main",
-            validate=lambda q: q in valid_queries,
+            validate=lambda q: q in valid_queries_dirs,
         )
 
         queries_path = queries_path_root / query_choice
         display_name = "UCX Assessment Results"
         dashboard_metadata = DashboardMetadata(display_name)
-        assessment_results = (
-            DashboardMetadata.from_path(queries_path)
-                .replace_database(catalog="hive_metastore", database=self._config.inventory_database)
+        assessment_results = dashboard_metadata.from_path(queries_path).replace_database(
+            catalog="hive_metastore", database=self._config.inventory_database
         )
 
         logger.info("Exporting assessment results....")
         results_path = assessment_results.export_to_zipped_csv(self._sql_backend, export_path)
+        print("results_path", results_path)
```

Comment:
Remove this print statement. If we are printing, we should use a logger and use the info category. I also think this is too verbose, we don't need to print it.

Reply:
Updated